tests/aws/services/stepfunctions/templates/services/services_templates.py

```diff
@@ -23,6 +23,9 @@ class ServicesTemplates(TemplateLoader):
     AWS_SDK_SFN_START_EXECUTION: Final[str] = os.path.join(
         _THIS_FOLDER, "statemachines/aws_sdk_sfn_start_execution.json5"
     )
+    AWS_SDK_SFN_START_EXECUTION_2: Final[str] = os.path.join(
+        _THIS_FOLDER, "statemachines/aws_sdk_sfn_start_execution_2.json5"
```

Comment:
nit: `_2` could be replaced with something a bit more descriptive like `_implicit_json_conversion` or something similar